utils/keys.go

```diff
@@ -0,0 +1,115 @@
+package utils
+
+import (
+	"encoding/pem"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+)
+
+// simple interface for the one function we need from the Storage interface
+type exportStore interface {
+	Get(string) ([]byte, error)
+	ListFiles() []string
+}
+
+type importStore interface {
+	Set(string, []byte) error
+}
+
+// ExportKeysByGUN exports all keys filtered to a GUN
+func ExportKeysByGUN(to io.Writer, s exportStore, gun string) error {
+	keys := s.ListFiles()
+	for _, k := range keys {
+		dir := filepath.Dir(k)
+		if dir == gun { // must be full GUN match
```

Comment:
I think we need to chomp off any part of the filepath preceding `<some_path>/<"trust_dir">/private` in order to match against a GUN